msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,135 @@
+          "dependsOn" : [
+              {"name" : "bots", "id" : "bot-id"}
+            ]
+        }
+      ]
+    }
+```
+
+For message extensions, you can optionally specify individual commands that require support for specific app capabilities. If those capabilities aren't supported in the runtime host, they won't be made available to the user (though all other commands will run).
+
+### Mutual dependencies
+
+Use the `mutualDependencies` array to group app capabilities that must load together in order to support their intended function. Each object in the array represents a set of mutually dependent app capabilities. The following JSON snippet shows a bot, static tab, message extension, and configurable tab that are mutually dependent on each other:
+
+```json
+    "elementRelationshipSet": {
+      "mutualDependencies" : [
+        [
+                {"name" : "bots", "id" : "bot-id"}, 
+                {"name" : "staticTabs", "id" : "staticTab-id"},
+                {"name" : "composeExtensions", "id" : "composeExtension-id"},
+                {"name" : "configurableTabs", "id": "configurableTab-id"}
+        ]
+      ]
+    },
+```
+
+## Specify runtime requirements for specific app capabilities (`requirementSet`)
+
+Within individual app capability definitions, you can specify specific TeamsJS runtime requirements using a [`requirementSet`](../resources/schema/manifest-schema-dev-preview.md#statictabsrequirementset). This ensures that the app capability only loads in Microsoft 365 hosts with support for the critical TeamsJS capabilities. The following JSON snippet shows a static tab that requires its host to support HTML dialogs invoked from tabs and bots: 
+
+```json
+    "staticTabs": [
+        {
+            "entityId": "idForPage",
+            "name": "Display name of tab",
+            "contentUrl": "https://contoso.com/content?host=msteams",
+            "contentBotId": "Specifies to the app that tab is an Adaptive Card Tab. You can either provide the contentBotId or contentUrl.",
+            "websiteUrl": "https://contoso.com/content",
+            "scopes": [
+                "personal"
+            ],
+            "requirementSet": {
+                "hostMustSupportFunctionalities": [
+                  {"name": "dialogUrl"},
+                  {"name": "dialogUrlBot"}
+                ]
+            }
+        }
+    ],
+```
+
+The following TeamsJS capabilities can be specified as runtime requirements for [`staticTabs`](../resources/schema/manifest-schema-dev-preview.md#statictabsrequirementset), [`composeExtensions`](../resources/schema/manifest-schema-dev-preview.md#composeextensionsrequirementset), and [`bots`](../resources/schema/manifest-schema-dev-preview.md#botsrequirementset):
+
+- HTML-based dialogs (`dialog.url`)
+- HTML-based dialogs for Bot Framework (`dialog.url.bot`)
+- Adaptive Card dialogs (`dialog.adaptiveCard`)
+- Adaptive Card dialogs for Bot Framework (`dialog.adaptiveCard.bot`)
+
+## See also
```

Comment:
@erikadoyle, can we link [Dialogs](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/what-are-task-modules) here?